core-bundle/src/Resources/contao/modules/ModuleLogin.php

```diff
@@ -54,9 +59,15 @@ public function generate()
 			return $objTemplate->parse();
 		}
 
+		$this->targetPath = '';
+
 		if (!$_POST && $this->redirectBack && ($strReferer = $this->getReferer()) != Environment::get('request'))
 		{
-			$_SESSION['LAST_PAGE_VISITED'] = $strReferer;
+			$this->targetPath = Environment::get('base').$strReferer;
```

Comment:
Does `_target_path` have to be absolute? Because now it can be set to any URL, whereas before it was limited to the same base path.

Reply:
I think it has to be this way, because the POST parameter is absolute (and always was)